<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Yolo-11" failures="2" tests="2" name="Retry(failed)" time="0.058" errors="0" timestamp="2022-02-23T20:45:48 IST">
  <testcase classname="testCases.SeleniumInterview" name="failedretry1" time="0.016">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<false> but was:<true>
at junit.framework.Assert.fail(Assert.java:50)
at junit.framework.Assert.failNotEquals(Assert.java:287)
at junit.framework.Assert.assertEquals(Assert.java:67)
at junit.framework.Assert.assertEquals(Assert.java:147)
at junit.framework.Assert.assertEquals(Assert.java:153)
at testCases.SeleniumInterview.failedretry1(SeleniumInterview.java:205)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- failedretry1 -->
  <testcase classname="testCases.SeleniumInterview" name="failedretry2" time="0.002">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<false> but was:<true>
at junit.framework.Assert.fail(Assert.java:50)
at junit.framework.Assert.failNotEquals(Assert.java:287)
at junit.framework.Assert.assertEquals(Assert.java:67)
at junit.framework.Assert.assertEquals(Assert.java:147)
at junit.framework.Assert.assertEquals(Assert.java:153)
at testCases.SeleniumInterview.failedretry2(SeleniumInterview.java:209)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- failedretry2 -->
</testsuite> <!-- Retry(failed) -->
